events.html

```diff
@@ -248,6 +248,34 @@ <h3 class="speaker-info">Works at DocOn Technologies</h3>
 					optimizing DOM manipulation and code
 				</li>
 			</ul>
```

Comment:
I think this should be removed, as you have closed it below

Reply:
it's GitHub file compare issue for closing "div" tag that's why it's showing on top.
if you check at end of code, it's not there.
I rechecked it to confirm.